packages/formiz-core/README.md

```diff
@@ -0,0 +1,27 @@
+# @formiz/core
+
+Core package for the [Formiz](https://formiz-react.com) library.
+
+## 📚 Full documentation
+
+Full documentation: [formiz-react.com](https://formiz-react.com)
+
+## Install
+
+Using npm:
+
+```
+npm install @formiz/core@alpha
```

Comment:
As we aim for v2, I don't think we will go through the READMEs later and that suggesting `alpha` is not a good thing.

Reply:
This will be update when v2 will be release, like the documentation where we notice `@alpha` too